weis/aeroelasticse/openmdao_openfast.py

```diff
@@ -1838,24 +1840,51 @@ def run_FAST(self, inputs, discrete_inputs, fst_vt):
         # Set initial rotor speed and pitch if the WT operates in this DLC and available,
         # otherwise set pitch to 90 deg and rotor speed to 0 rpm when not operating
         # set rotor speed to rated and pitch to 15 deg if operating
+        if self.options['modeling_options']['Level3']['from_openfast']:
+            try:
+                reg_traj = self.options['modeling_options']['Level3']['regulation_trajectory']
+                data = load_yaml(reg_traj)
+                cases = data['cases']
+                n_ws = len(cases)
+                U_interp = [case["configuration"]["wind_speed"] for case in cases]
+                pitch_interp = [case["configuration"]["pitch"] for case in cases]
+                rot_speed_interp = [case["configuration"]["rotor_speed"] for case in cases]
+                Ct_aero_interp = [case["outputs"]["integrated"]["ct"] for case in cases]
+            except:
```

Comment:
Is this your final implementation @ptrbortolotti?  I'm a little wary of the try/except with all those nested arrays.  `if reg_traj = unused:` might avoid some pain later.

Reply:
[ee279ab](https://github.com/WISDEM/WEIS/pull/261/commits/ee279abf3d6deb61aac33e21f2e33f4d0b4a5611)

replaced with if else